documents/extension/funding/funding.go

```diff
@@ -0,0 +1,33 @@
+package funding
+
+import (
+	"reflect"
+
+	clientfundingpb "github.com/centrifuge/go-centrifuge/protobufs/gen/go/funding"
+)
+
+// Data is the default funding extension schema
+type Data struct {
+	FundingId             string `json:"funding_id,omitempty" attr:"string"`
+	Amount                string `json:"amount,omitempty" attr:"string"`
+	Apr                   string `json:"apr,omitempty" attr:"string"`
+	Days                  string `json:"days,omitempty" attr:"string"`
+	Fee                   string `json:"fee,omitempty" attr:"string"`
+	RepaymentDueDate      string `json:"repayment_due_date,omitempty" attr:"string"`
+	RepaymentOccurredDate string `json:"repayment_occurred_date,omitempty" attr:"string"`
+	RepaymentAmount       string `json:"repayment_amount,omitempty" attr:"string"`
+	Currency              string `json:"currency,omitempty" attr:"string"`
+	NftAddress            string `json:"nft_address,omitempty" attr:"string"`
+	PaymentDetailsId      string `json:"payment_details_id,omitempty" attr:"string"`
+}
+
+func (f *Data) initFundingFromData(data *clientfundingpb.FundingData) {
```

Comment:
Why use reflection here in a specific document extension? Are we planning on adding more extensions soon? in which case then we might want to have its own package with common init functions.

Reply:
I think it is better to write the converter for this instead.